packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -138,15 +138,17 @@ library ChainAssertions {
             require(address(messenger.otherMessenger()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "CHECK-L1XDM-30");
             require(address(messenger.PORTAL()) == _contracts.OptimismPortal, "CHECK-L1XDM-40");
             require(address(messenger.portal()) == _contracts.OptimismPortal, "CHECK-L1XDM-50");
-            require(address(messenger.superchainConfig()) == _contracts.SuperchainConfig, "CHECK-L1XDM-60");
+            require(
+                address(messenger.systemConfig().superchainConfig()) == _contracts.SuperchainConfig, "CHECK-L1XDM-60"
+            );
             bytes32 xdmSenderSlot = _vm.load(address(messenger), bytes32(uint256(204)));
             require(address(uint160(uint256(xdmSenderSlot))) == Constants.DEFAULT_L2_SENDER, "CHECK-L1XDM-70");
         } else {
             require(address(messenger.OTHER_MESSENGER()) == address(0), "CHECK-L1XDM-80");
             require(address(messenger.otherMessenger()) == address(0), "CHECK-L1XDM-90");
             require(address(messenger.PORTAL()) == address(0), "CHECK-L1XDM-100");
             require(address(messenger.portal()) == address(0), "CHECK-L1XDM-110");
-            require(address(messenger.superchainConfig()) == address(0), "CHECK-L1XDM-120");
+            require(address(messenger.systemConfig().superchainConfig()) == address(0), "CHECK-L1XDM-120");
```

Comment:
Same as above, check system config. This will break because `systemConfig` is `address(0)` which means `superchainConfig()` call will not work.